### PR TITLE
fix issue#1

### PR DESCRIPTION
**Pull request by MirrorShih**
add new environment variable MESSAGE_CHANNEL_ID for user to choose which channel to listen

### CODE REVIEW

File: d2l.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 discordbot_token = os.environ['DISCORDBOT_TOKEN']
 lotify_token = os.environ['LOTIFY_TOKEN']
 discord_webhook_id = int(os.environ['DISCORD_WEBHOOK'].split('/')[-2])
+message_channel_id=None if os.environ.get('MESSAGE_CHANNEL_ID') is None else int(os.environ.get('MESSAGE_CHANNEL_ID'))
```
Comment (CA-Lee) [inline d2l.py]: get 可以指定預設值，像這樣 `os.environ.get('MESSAGE_CHANNEL_ID', None)`
you may use default value with `get` like this `os.environ.get('MESSAGE_CHANNEL_ID', None)`

**Review by CA-Lee — COMMENTED** [review]
Hi, 感謝你的貢獻，有些地方我覺得可以更精簡，給你參考。另外，請問你有測試過這份 code 了嗎?
Hi, thanks for contributing. It looks good to me, with some micro suggestions for you. Besides, have you tested this code?

File: d2l.py (hunk 1 above)
Comment (MirrorShih) [reply d2l.py]: OK, I change it to `os.environ.get('MESSAGE_CHANNEL_ID', None)` , and yes, I had tested the code

**Comment by MirrorShih** [comment]
I push a new commit that I add a deploy button in README.md, user can easily deploy it to Heroku now

**Review by CA-Lee — APPROVED** [review]

**Comment by CA-Lee** [comment]
All looks good, thank you for contributing!

**Pull request merged**